lib/order_reporting.rb

```diff
@@ -1,12 +1,24 @@
 require 'order_reporting/engine'
 require 'order_reporting/report'
+require 'order_reporting/schedulers/delayed_job_scheduler'
+require 'order_reporting/queries/latest_orders_query'
```

Comment:
Should there be a dependancy on that query code within here? The query will be defined by the parent applications, so question whether the library should know about it.